Libraries/Components/WebView/WebView.android.js

```diff
@@ -197,6 +205,25 @@ class WebView extends React.Component {
     saveFormDataDisabled: PropTypes.bool,
 
     /**
+     * Override the native component used to render the WebView. Enables a custom native
+     * WebView which uses the same JavaScript as the original WebView.
+     */
+    nativeConfig: PropTypes.shape({
+      /*
+       * The native component used to render the WebView.
+       */
+      component: PropTypes.any,
+      /*
+       * Set props directly on the native component WebView. Enables custom props which the
+       * original WebView doesn't pass through.
+       */
+      props: PropTypes.object,
+      /*
+       * Set the ViewManager to use for communcation with the native side.
+       * @platform ios
+       */
+      viewManager: PropTypes.object,
+    }),
      * Used on Android only, controls whether the given list of URL prefixes should
```

Comment:
Here's why the PR hasn't landed. I'll see if I can fix it on our side, no need to update your PR.

Reply:
Alright, noted. Seems to be caused by an earlier merge. Thanks! 👍